sdk-cpp/tests/CMakeLists.txt

```diff
@@ -3,14 +3,20 @@
 find_package(Boost COMPONENTS filesystem program_options REQUIRED)
 find_package(GTest REQUIRED)
 
+if(${DO_PLATFORM_ID} MATCHES ${DO_PLATFORM_ID_LINUX})
```

Comment:
>MATCHES [](start = 21, length = 7)

Curious why equality doesn't work here

Reply:
Either's fine, equal is probably better to use here since matches performs a regex match

---
In reply to: [605214352](https://github.com/microsoft/do-client/pull/18#discussion_r605214352) [](ancestors = 605214352)

Reply:
Agreed. I'm still seeing matches here though.

---
In reply to: [605235290](https://github.com/microsoft/do-client/pull/18#discussion_r605235290) [](ancestors = 605235290,605214352)